config/variables.cfg

```diff
@@ -99,6 +99,9 @@ variable_filter_name: "filter"
 variable_light_enabled: True
 variable_light_pin_name: "light"
 
+variable_light_printing: 30
+variable_light_end_print: 5
+
```

Comment:
This a good idea !
Can you first remove the first empty line to make it a block of `variable_light_xxxx` parameters ?

Also, maybe this could be extended a little bit to have variables also for the start_print phase like:
```
variable_light_intensity_start_print: 100
variable_light_intensity_printing: 30
variable_light_intensity_end_print: 5
```
I also suggest to add the "intensity" word in the naming to better understand the effect of this parameter.

Reply:
I like it.
Will make this change